### PR TITLE
[ Selenium ] Cover the "Python" language server functionality by selenium test 

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Cover the "Python" language server functionality by selenium test

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/11218

### CODE REVIEW

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java [hunk 1]
```diff
@@ -378,6 +378,51 @@ public String getVisibleTextFromEditor() {
...
+    if (0 == editorLine) {
+      String errorMessage =
+          format(
+              "Specified line: \"%s\" does not exist. The editor numeration starts from \"1\". "
+                  + "Please specify correct line.",
+              editorLine);
+      throw new ArrayIndexOutOfBoundsException(errorMessage);
+    }
+
+    final int lineIndex = editorLine - 1;
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java]: Could be useful as a separate method of CodenvyEditor: `getVisibleLineFromEditor(lineNumber)`.

**Comment by artaleks9** [comment]
Please, fix the method  _clickOnActionNodeWithText(String visibleText)_ in the _AssistantFindPanel_ on the :
```
  public void clickOnActionNodeWithText(String visibleText) {
    for (int i = 0; i < getActionNodesCount(); i++) {
      if (getActionNodeText(i).contains(visibleText)) {
        getActionNode(i).click();
        return;
      }
    }

    String exceptionMessage =
        String.format("Item with expected visible text: \"%s\" has not been detected", visibleText);
    throw new RuntimeException(exceptionMessage);
  }

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/python/PythonAssistantFeatureTest.java [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    URL resource = PythonFileEditingTest.class.getResource("/projects/console-python3-simple");
+    testProjectServiceClient.importProject(
+        workspace.getId(), Paths.get(resource.toURI()), PROJECT_NAME, ProjectTemplates.PYTHON);
+    ide.open(workspace);
+    projectExplorer.waitProjectExplorer();
+    projectExplorer.openItemByPath(PROJECT_NAME);
+  }
+
+  @Test
+  public void checkCommenting() {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/python/PythonAssistantFeatureTest.java]: Let's be more specific in test naming: `checkCodeCommentFeature`

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/python/PythonAssistantFeatureTest.java [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+    editor.setCursorToLine(16);
+    editor.launchCommentCodeFeature();
+    editor.waitTextIntoEditor(EXPECTED_COMMENTED_CODE);
+    editor.launchCommentCodeFeature();
+    editor.waitTextIntoEditor(EXPECTED_CODE_BEFORE_COMMENTING);
+  }
+
+  @Test
+  public void checkRenaming() {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/python/PythonAssistantFeatureTest.java]: Let's be more specific in test naming: `checkRenameFeature`

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/python/PythonAssistantFeatureTest.java [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+import org.eclipse.che.selenium.pageobject.Consoles;
+import org.eclipse.che.selenium.pageobject.FindReferencesConsoleTab;
+import org.eclipse.che.selenium.pageobject.Ide;
+import org.eclipse.che.selenium.pageobject.Menu;
+import org.eclipse.che.selenium.pageobject.ProjectExplorer;
+import org.openqa.selenium.TimeoutException;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class PythonAssistantFeatureTest {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/python/PythonAssistantFeatureTest.java]: Did you mean `PythonAssistantFeaturesTest` class name?

**Review by artaleks9 — APPROVED** [review]

**Review by dmytro-ndp — APPROVED** [review]

**Review by musienko-maxim — APPROVED** [review]

**Pull request merged**